resources/views/organizations/show.blade.php

```diff
@@ -56,10 +57,10 @@ class="mb-3 block rounded-xl bg-black/4 p-4 py-2 text-lg backdrop-blur-lg transi
 
         <div class="col-span-2 mt-8 md:mt-0 xl:col-span-3">
             @if ($organization->sites->count() === 0)
-                <img src="{{ $organization->image }}" class="rounded-md border" />
+                <img style="view-transition-name: no-site-{{ $organization->id }}" loading="lazy" src="{{ $organization->image }}" class="rounded-md border" />
             @else
                 @foreach ($organization->sites as $site)
-                    <div id="site-{{ $site->slug }}" class="mb-10">
+                    <div id="site-{{ $site->slug }}" class="mb-10" style="view-transition-name: main-site-{{ $site->id }}">
```

Comment:
Can we update these to use the site->slug instead? Just added that recently so we don't have to use the integer ID inline

Reply:
Done